app-editors/vis/vis-0.9-r1.ebuild

```diff
@@ -72,14 +76,15 @@ src_prepare() {
 }
 
 src_configure() {
-	./configure \
-		--prefix="${EPREFIX}"/usr \
-		--docdir="${EPREFIX}"/usr/share/doc/${PF} \
-		--disable-lpeg-static \
-		$(use_enable lua) \
-		$(use_enable ncurses curses) \
-		$(use_enable selinux) \
-		$(use_enable tre) || die
+	local myeconfargs=(
+		--disable-lpeg-static
+		$(use_enable lua)
+		$(use_enable ncurses curses)
+		$(use_enable selinux)
+		$(use_enable tre)
+	)
+
+	econf "${myeconfargs[@]}"
```

Comment:
`econf` is designed for autoconf, which vis isn't using.

Reply:
And if you follow the link for details:

https://devmanual.gentoo.org/ebuild-writing/functions/src_configure/configuring/index.html#econf-options
> econf is designed to work with configure scripts generated by GNU Autoconf.

Reply:
lanodan is correct.